src/ui/layout/tree-item.vue

```diff
@@ -142,6 +145,9 @@ export default {
         },
         resetTreeHere() {
             this.$emit('resetTree', this.node);
+        },
+        setContextClickEnabled(contextClickEnabled) {
```

Comment:
Since contextClickEnabled exists on the mixin that's included, do we need this? Shouldn't it just react to the changes? I didn't see it in grid or list views which use the same mixin, just curious.